app/services/analytics_events.rb

```diff
@@ -777,6 +777,7 @@ def idv_phone_confirmation_otp_resent(
   # @param [String] country_code country code of phone number
   # @param [String] area_code area code of phone number
   # @param [Boolean] rate_limit_exceeded whether or not the rate limit was exceeded by this attempt
+  # @param [String] phone_fingerprint
```

Comment:
what about adding some documentation of which fingerprint?
```suggestion
  # @param [String] phone_fingerprint the hmac fingerprint of the phone number formatted as e164
```